docs/threat-model.md

```diff
@@ -114,7 +114,147 @@ For each asset we define what could be impacted in respect to its Confidentialit
 
 ## Threats
 
-Notable threats to the system that could impact assets.
+The Leios protocol may be vulnerable to the following categories of threats. Each category represents fundamental attack surfaces that stem from the protocol's design and operation.
+
+### Praos dependency and VRF-based eligibility
```

Comment:
The fact that the threats are ordered differently in the Readme and in the threat model make it harder to follow.